From 77977396865f4099dff7143c703301ccea52a276 Mon Sep 17 00:00:00 2001
From: Nalin Dahyabhai <nalin@redhat.com>
Date: Tue, 14 Jun 2016 15:57:16 -0400
Subject: [PATCH] Comment/whitespace fixup

---
 src/ipa.c    | 2 +-
 src/tdbush.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ipa.c b/src/ipa.c
index 72cdda6b07ea5a4850fb404497196c46a6bbbd6d..f2736c6f37948df902b65157480fc0c29ec58c3e 100644
--- a/src/ipa.c
+++ b/src/ipa.c
@@ -367,7 +367,7 @@ submit:
 	if (profile != NULL) {
 		cm_submit_x_add_named_arg_s(ctx, "profile_id", profile);
 	}
-	/* Add the requested CA named argument. */
+	/* Add the requested CA issuer named argument. */
 	if (issuer != NULL) {
 		cm_submit_x_add_named_arg_s(ctx, "ca", issuer);
 	}
diff --git a/src/tdbush.c b/src/tdbush.c
index 05a503e06a553c566dcff5e053cbd8aa16c20f14..631da3ed2bbb1f6828d576760299ad51d7e41923 100644
--- a/src/tdbush.c
+++ b/src/tdbush.c
@@ -1566,7 +1566,7 @@ base_add_request(DBusConnection *conn, DBusMessage *msg,
 					  cm_tdbusm_dict_s);
 	if (param != NULL) {
 		new_entry->cm_template_issuer = maybe_strdup(new_entry,
-							      param->value.s);
+							     param->value.s);
 	}
 	param = cm_tdbusm_find_dict_entry(d,
 					  CM_DBUS_PROP_TEMPLATE_CHALLENGE_PASSWORD,
-- 
2.9.0

