From 43392d48924d6d50ef2712947ddea424e723e171 Mon Sep 17 00:00:00 2001
From: Trevor Vaughan <tvaughan@onyxpoint.com>
Date: Tue, 27 Mar 2018 09:28:28 -0400
Subject: [PATCH 23/25] Updates per Feedback

Ref: #89
---
 src/scepgen-o.c | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/src/scepgen-o.c b/src/scepgen-o.c
index 07c2b8b..05fc437 100644
--- a/src/scepgen-o.c
+++ b/src/scepgen-o.c
@@ -457,7 +457,7 @@ cm_scepgen_o_cooked(struct cm_store_ca *ca, struct cm_store_entry *entry,
 			_exit(1);
 		}
 
-		cm_log(1, "SCEP cipher authoritatively set to: '%s'\n", scep_cipher);
+		cm_log(1, "SCEP cipher set from configuration to: '%s'\n", scep_cipher);
 	}
 	else {
 		cipher = cm_prefs_nocipher;
@@ -527,11 +527,11 @@ cm_scepgen_o_cooked(struct cm_store_ca *ca, struct cm_store_entry *entry,
 			digest = cm_prefs_md5;
 		}
 		else {
-			cm_log(1, "Option 'scep_digest' must be one of AES256, AES192, AES128, DES3, or DES. Got '%s'\n", scep_digest);
+			cm_log(1, "Option 'scep_digest' must be one of SHA512, SHA384, SHA256, SHA1, or MD5. Got '%s'\n", scep_digest);
 			_exit(1);
 		}
 
-		cm_log(1, "SCEP digest authoritatively set to: '%s'\n", scep_digest);
+		cm_log(1, "SCEP digest set from configuration to: '%s'\n", scep_digest);
 	}
 	else {
 		pref_digest = cm_prefs_preferred_digest();
@@ -578,7 +578,8 @@ cm_scepgen_o_cooked(struct cm_store_ca *ca, struct cm_store_entry *entry,
 			}
 		}
 		if (digest == cm_prefs_nodigest) {
-			/* Per the latest Draft RFC */
+			/* Per SCEP RFC draft-gutmann-scep-10 - March 1, 2018 */
+			/* https://www.ietf.org/id/draft-gutmann-scep-10.txt  */
 			cm_log(1, "Could not determine supported CA capabilities, using SHA256.\n");
 			digest = cm_prefs_sha256;
 		}
-- 
1.8.3.1

