commit 70ceb7e9b55a8280e793861204ffbcf8388f50f0
Author: Nalin Dahyabhai <nalin@redhat.com>
Date:   Tue Jan 8 18:31:08 2013 -0500

    fix a crash in the self-test

diff --git a/tests/tools/iterate.c b/tests/tools/iterate.c
index 63874da..1c57662 100644
--- a/tests/tools/iterate.c
+++ b/tests/tools/iterate.c
@@ -38,11 +38,16 @@ wait_to_read(int fd)
 {
 	fd_set rfds;
 	struct timeval tv;
-	FD_ZERO(&rfds);
-	FD_SET(fd, &rfds);
-	tv.tv_sec = 1;
-	tv.tv_usec = 0;
-	select(fd + 1, &rfds, NULL, NULL, &tv);
+
+	if (fd >= 0) {
+		FD_ZERO(&rfds);
+		FD_SET(fd, &rfds);
+		tv.tv_sec = 1;
+		tv.tv_usec = 0;
+		select(fd + 1, &rfds, NULL, NULL, &tv);
+	} else {
+		sleep(1);
+	}
 }
 
 struct cm_context {
