commit ce297d93706f2022fc22f8969aea3c9039933e49
Author: Nalin Dahyabhai <nalin@redhat.com>
Date:   Tue Jan 8 14:57:01 2013 -0500

    reduce the number of "newly-added" resume states

diff --git a/src/iterate.c b/src/iterate.c
index 8dc3206..535fff7 100644
--- a/src/iterate.c
+++ b/src/iterate.c
@@ -168,22 +168,25 @@ cm_entry_reset_state(struct cm_store_entry *entry)
 	case CM_NEWLY_ADDED:
 		break;
 	case CM_NEWLY_ADDED_START_READING_KEYINFO:
+		entry->cm_state = CM_NEWLY_ADDED;
 		break;
 	case CM_NEWLY_ADDED_READING_KEYINFO:
-		entry->cm_state = CM_NEWLY_ADDED_START_READING_KEYINFO;
+		entry->cm_state = CM_NEWLY_ADDED;
 		break;
 	case CM_NEWLY_ADDED_NEED_KEYINFO_READ_TOKEN:
-		entry->cm_state = CM_NEWLY_ADDED_START_READING_KEYINFO;
+		entry->cm_state = CM_NEWLY_ADDED;
 		break;
 	case CM_NEWLY_ADDED_NEED_KEYINFO_READ_PIN:
-		entry->cm_state = CM_NEWLY_ADDED_START_READING_KEYINFO;
+		entry->cm_state = CM_NEWLY_ADDED;
 		break;
 	case CM_NEWLY_ADDED_START_READING_CERT:
+		entry->cm_state = CM_NEWLY_ADDED;
 		break;
 	case CM_NEWLY_ADDED_READING_CERT:
-		entry->cm_state = CM_NEWLY_ADDED_START_READING_CERT;
+		entry->cm_state = CM_NEWLY_ADDED;
 		break;
 	case CM_NEWLY_ADDED_DECIDING:
+		entry->cm_state = CM_NEWLY_ADDED;
 		break;
 	case CM_INVALID:
 		/* not reached */
